packages/core/src/node/build.ts

```diff
@@ -112,6 +113,7 @@ export async function renderPages(
   const ssrBundlePath = join(outputPath, 'ssr', 'main.cjs');
   try {
     const { default: fs } = await import('@rspress/shared/fs-extra');
+    const { version } = await import('../../package.json');
```

Comment:
Have you test the `../../package.json` path?

Reply:
Yes, I've tested it and it works in deploy preview.

![1704677478295](https://github.com/web-infra-dev/rspress/assets/44596995/ae9e6381-e5e7-45e8-8d58-e386f93a4d30)